src/pencilkit.cs

```diff
@@ -97,12 +115,20 @@ interface PKCanvasView : PKToolPickerObserver {
 		[Export ("drawingGestureRecognizer")]
 		UIGestureRecognizer DrawingGestureRecognizer { get; }
 
+		[Introduced (PlatformName.iOS, 13, 0, message: "Use 'drawingPolicy' property instead.")]
+		[Deprecated (PlatformName.iOS, 14, 0, message: "Use 'drawingPolicy' property instead.")]
 		[Unavailable (PlatformName.UIKitForMac), Advice ("This API is not available when using UIKit on macOS.")]
 		[Export ("allowsFingerDrawing")]
 		bool AllowsFingerDrawing { get; set; }
+
+		[iOS (14, 0)]
+		[Unavailable (PlatformName.UIKitForMac), Advice ("This API is not available when using UIKit on macOS.")]
```

Comment:
^ @dalexsoto seems sharpie still use the old platform name for Unavailable

Reply:
@spouliot should I go through and update all of the changes that use `PlatformName.UIKitForMac` to use Catalyst instead?

Reply:
@whitneyschmidt for this file ? yes, it can be done inside this PR

For all files ? this should be a separate PR and might conflict with other people work on bindings. It's best to do it gradually :)